views/js/dataProvider/reportsDataProvider.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA ;
+ */
+
+define([
+    'util/url',
+    'core/request',
+    'json!./data.json'
+], function (urlHelper, request, data) {
+
+    const reportCategories = {
+        configuration: [
```

Comment:
In order to avoid changed in that file in case if check was added/removed my suggestion is to add category id and renderer id for each checker.